Tests/test_geo.py

```diff
@@ -3165,6 +3164,91 @@ def test_soft_ex_affy_chp(self):
+                                          "Sample_submitter_name: Donna,G,Albertson\n\n"
+                                          "Sample_submitter_phone: 415 502-8463\n\n"
+                                          "Sample_target_source1: Cell line GM05296\n\n"
+                                          "Sample_target_source2: normal male reference genomic DNA\n\n"
+                                          "Sample_title: CGH_Albertson_GM05296-001218\n\n"
+                                          "Sample_type: dual channel genomic\n\n"
+                                          "Column Header Definitions\n"
+                                          "    ID_REF: Unique row identifier, genome position o\n"
+                                          "    rder\n\n"
+                                          "    LINEAR_RATIO: Mean of replicate Cy3/Cy5 ratios\n\n"
+                                          "    LOG2STDDEV: Standard deviation of VALUE\n\n"
+                                          "    NO_REPLICATES: Number of replicate spot measurements\n\n"
+                                          "    VALUE: aka LOG2RATIO, mean of log base 2 of LIN\n"
+                                          "    EAR_RATIO\n\n"
+                                          "0: ID_REF	VALUE	LINEAR_RATIO	LOG2STDDEV	NO_REPLICATES\t\n"
+                                          "1: 1		1.047765	0.011853\t3\t\n"
+                                          "2: 2\t\t\t\t0\t\n"
+                                          "3: 3	0.008824	1.006135	0.00143\t3\t\n"
+                                          "4: 4	-0.000894	0.99938	0.001454\t3\t\n"
+                                          "5: 5	0.075875	1.054	0.003077	3\t\n"
+                                          "6: 6	0.017303	1.012066	0.005876	2\t\n"
+                                          "7: 7	-0.006766	0.995321	0.013881	3\t\n"
+                                          "8: 8	0.020755	1.014491	0.005506	3\t\n"
+                                          "9: 9	-0.094938	0.936313	0.012662	3\t\n"
+                                          "10: 10	-0.054527	0.96291	0.01073	3\t\n"
+                                          "11: 11	-0.025057	0.982782	0.003855	3\t\n"
+                                          "12: 12				0\t\n"
+                                          "13: 13	0.108454	1.078072	0.005196	3\t\n"
+                                          "14: 14	0.078633	1.056017	0.009165	3\t\n"
+                                          "15: 15	0.098571	1.070712	0.007834	3\t\n"
+                                          "16: 16	0.044048	1.031003	0.013651	3\t\n"
+                                          "17: 17	0.018039	1.012582	0.005471	3\t\n"
+                                          "18: 18	-0.088807	0.9403	0.010571	3\t\n"
+                                          "19: 19	0.016349	1.011397	0.007113	3\t\n"
+                                          "20: 20	0.030977	1.021704	0.016798	3\t\n")
+
```

Comment:
You would still need the ``\t`` for tabs, but would triple quotes make it easier to express this without the explicit ``\n`` for new lines - and perhaps avoid any indentation at all?